### DISCUSSION TITLE
Allow custom entry list views to be saved as a default view for member groups/roles

### DISCUSSION THREAD

**Discussion by aasun** [ExpressionEngine/ExpressionEngine#3298] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
In the Channel Entry list view, EE 6 and EE 7 currently have the ability to choose what columns to display, and what order to display them..  It would be excellent to be able to set/save that as a default view per member group/role…. Very similar to how we can currently save Channel Publish layouts per Member group/role.

**Describe the solution you'd like**
Allow any super admin to create an entry list view per channel, and save that view as the default (starting) view for any member group.  A consideration would be that members could still create their own custom views..  So, maybe with this option, if a super admin creates a saved view, then other CP users could not create their own? (CP members can't create their own Publish layouts).. Or, perhaps, the super admin just creates the "default" entry list view, and CP members can still create their own to customize that.

**Describe alternatives you've considered**

**Teachability, Documentation, Adoption, Migration Strategy**
If you can, explain how users will be able to use this and possibly write out a version the docs. Some of the best features are born from writing documentation first. Consider making a documentation pull request from a feature branch here: https://github.com/ExpressionEngine/ExpressionEngine-User-Guide.

- [ ] I am capable and would like to work on implementation of this feature if it is considered.

**Comment by pyrobob83** [comment]
+1  :D
